src/routes/collection/+page.svelte

```diff
@@ -0,0 +1,474 @@
+<svelte:head>
+    <title>Lovat Collection | Lovat</title>
+</svelte:head>
+
+
+<section class="front-page">
+    <div class="content-wrapper">
+        <div class="header"> 
+            <img class="icon" src="/assets/img/collection-icon.png" alt="Screenshot of Lovat's Collection icon.">
+            <h1>Lovat Collection</h1>
+        </div>
+        <p>Watch our training video:</p>
+        <div class="video-description">
+            <br>
+            <p>This video covers every single feature in Lovat Collection including onboarding, settings, all the game screens, and more. It is crucial that every single scouter watches this video in its entirety. All the resources below are covered in the video, and are just there to serve as reminders, not as alternate training materials.</p>
+        </div>
+    </div>
+</section>
+
+<section class="onboarding">
+    <div class="onboarding-content">
+        <h1>Onboarding</h1>
+        <div class="phone-row">
+            <div class="phone-column">
+                <div class="phone centered">
```

Comment:
The phones seem repetitive and difficult to style. I can make a component to make it easier.